cocos/core/utils/pool.ts

```diff
@@ -85,7 +85,7 @@ export class Pool<T> {
      * 当前可用对象数量，一开始默认是 0，随着对象的回收会逐渐增大，最大不会超过调用构造函数时指定的 size。
      * @default 0
      */
-    public count: number;
+    public count: number = 0;
```

Comment:
Why not use declare.

Reply:
Only use `declare` when properties are initialized in `constructor`. Otherwise, it doesn't need to use `declare` keyword.

Reply:
It is initialized in constructor.

Reply:
![WeChat471f440eb0f72b454be6f5da9e003dac](https://github.com/user-attachments/assets/b03527db-5db0-4712-9e8d-0531f4bcfed8)